static/sass/styles.scss

```diff
@@ -349,3 +349,24 @@ h3 {
   top: -50%;
   width: 40px;
 }
+
+.p-modal-tooltip {
+  display: inline-block;
+  position: relative;
+}
+
+.p-modal-tooltip__body {
+  background-color: $color-x-light;
+  box-shadow: $box-shadow;
+  display: none;
+  left: -1rem;
+  max-width: 420px;
+  padding: $sp-medium;
+  position: absolute;
+  top: 100%;
+  z-index: 10;
+}
+
+.p-modal-tooltip:hover .p-modal-tooltip__body {
```

Comment:
I don't think we want it on `hover`. It feels on click is cleaner, we don't have random stuff poping up when i move my mouse around. @danielmutis ?

Reply:
Yes I agree @tbille only on click is better.